ui/component/publishAdditionalOptions/view.jsx

```diff
@@ -18,9 +19,21 @@ type Props = {
 };
 
 function PublishAdvanced(props: Props) {
-  const { language, name, licenseType, otherLicenseDescription, licenseUrl, updatePublishForm } = props;
+  const {
+    language,
+    defaultLanguage,
+    name,
+    licenseType,
+    otherLicenseDescription,
+    licenseUrl,
+    updatePublishForm,
+  } = props;
   const [hideSection, setHideSection] = usePersistedState('publish-advanced-options', true);
 
+  useEffect(() => {
+    updatePublishForm({ language: defaultLanguage });
```

Comment:
this can overwrite an already set language

Reply:
What do you mean? It should take into account the original language that was set: https://github.com/lbryio/lbry-desktop/pull/3481/files/a4cdca9bbc2ff74b9c5bc31012fdf04efce63ade#diff-c104ac6e73d6345cf832072a703f7051R13

Reply:
@kauffj